tests/deepspeed/test_deepspeed.py

```diff
@@ -1051,3 +1051,9 @@ def test_lr_scheduler(self):
         ]
         with patch_environment(omp_num_threads=1):
             execute_subprocess_async(cmd)
+
+    def test_zero3_integration(self):
+        self.test_file_path = self.test_scripts_folder / "test_zero3_integration.py"
+        cmd = ["torchrun", "--standalone", "--nnodes=1", "--nproc-per-node=2", self.test_file_path]
```

Comment:
Use `accelerate launch` here instead as with the rest of how these scripts launch please

```suggestion
        cmd = ["accelerate", "launch", "--num_processes=2", "--num_machines=1", self.test_file_path]
```

Reply:
updated